### PR TITLE
Update download site to match README.md

### PR DESCRIPTION
**Pull request by glowiak**

### CODE REVIEW

File: src/download.md [hunk 1]
```diff
@@ -96,4 +98,43 @@ An RPM package is available on [COPR](https://copr.fedorainfracloud.org/coprs/se
 sudo dnf copr enable sentry/polymc
```
Comment (DioEgizio) [inline src/download.md:98]: this needs to be updated too

Comment (glowiak) [reply src/download.md:98]: oh yes i forgot to

Comment (glowiak) [reply src/download.md]: done

**Comment by ZekeSmith** [comment]
Looks good I am probably going to be working on the redesign soon but having this infomation will be useful

**Pull request merged**